scraper/src/index.py

```diff
@@ -114,3 +125,4 @@ def run_config(config):
     from os import environ
 
     run_config(environ['CONFIG'])
+    
```

Comment:
Could you remove the empty space in this line?
Plus the file should end with a new line 🙂